libs/vkd3d/acceleration_structure.c

```diff
@@ -19,6 +19,8 @@
 #define VKD3D_DBG_CHANNEL VKD3D_DBG_CHANNEL_API
 #include "vkd3d_private.h"
 
+#define RT_TRACE TRACE
```

Comment:
do we plan to decouple this at any point or is this just meant to be changed for local debugging?

Reply:
Mostly just for local debug. We might want to consider a separate debug channel, but it's only really useful when doing deep dives.